servers/references-server/src/main.rs

```diff
@@ -11,14 +13,13 @@ use tower_lsp::lsp_types::*;
 use tower_lsp::{Client, LanguageServer, LspService, Server};
 
 struct Backend {
-    client: Client,
     references_provider: Arc<Mutex<ReferencesProvider>>,
```

Comment:
are you sure it is a good idea to remove client (won't we need it in the future ?)

Reply:
We'll zimply re-add it if we need to, it is a simple line to add